### PR TITLE
Update italian language

### PR DESCRIPTION
**Pull request by bovirus**
@aurelienpierre 

General revison of language file for not right terms

Files -> Files
Click -> Clic
Con successo -> Corretamente
Mostra -> Visualizza

### CODE REVIEW

**Comment by TurboGit** [comment]
@mmardegan : Can you have a look and approve or not the change here? TIA.

File: po/it.po [hunk 1]
```diff
@@ -514,20 +505,20 @@ msgstr "Bordo intorno all'immagine in modalità Camera Oscura"
...
 #: ../build/bin/preferences_gen.h:4639
 msgid ""
-"process the image in darkroom mode with a small border. set to 0 if you "
-"don't want any border."
+"process the image in darkroom mode with a small border. set to 0 if you don't "
+"want any border."
 msgstr ""
-"Elabora l'immagine in modalità Camera Oscura con un bordo piccolo. Imposta a "
-"0 se non vuoi alcun bordo."
+"Elabora l'immagine in modalità 'Camera oscura' con un bordo piccolo. \n"
```
Comment (mmardegan) [inline po/it.po:515]: Per convenzione abbiamo concordato di usare Camera Oscura (senza virgolette) e Tavolo Luminoso con la maiuscola per identificare i rispettivi moduli. L'apostrofo non mi pare corretto.

**Comment by mmardegan** [comment]
@bovirus Would you like to contribute to the italian translation? We are using this repository and branch for the next update https://github.com/mmardegan/darktableITA/tree/Traduzione_4.0

**Comment by bovirus** [comment]
@mmardegan

Could you explain the reason of your fork?

Original darktable project has already the structiure and file for italian langauge, 
Infact I posted the PR for language update in this repository.

Why two different repositry for same project?

**Comment by mmardegan** [comment]
Maybe for the same reason that you fork it: why you fork it?
The repository that I linked is used to coordinate the italian translation using git: is used to coordinate the work, verify each other commits, discuss and agree a shared and consistent dictionary and finally make a single and complete pull request to the main project.
Would you like to contribute? We are glad to work in team.

**Comment by mmardegan** [comment]
> @mmardegan : Can you have a look and approve or not the change here? TIA.

Not completely: some modification are not compliant. @TurboGit  please refuse the PR. Thank you @bovirus , we will integrate some of your suggested modification in the next translation: this way is easiest than correct all the modifications (that are mainly text format).

File: po/it.po (hunk 1 above)
Comment (bovirus) [reply po/it.po:515]: Una regola base della grammatica italiana impone che le lettere maiuscole si usano solo per la prima lettera di una frase ad eccezione dei nomi propri.
Sono in inglese che usano le lettere maiuscole per ogni parola. Ma non vale per l'italiano.
Quindi Camera Oscura non va bene in italiano ma dovrebebe essere Camera oscura.
Idem per Tavolo luminoso.
L'apostrofo prima e dopo indica nelle descrizoni il riferimento ad una specifica funzione/opzione.

**Comment by bovirus** [comment]
@mmardegan 

I'm not agree that some mdoification are not compliant.
Could you please provide in details which of my changes are not compliant?

As I wrote before your "assumption" in current tranalstion, for me as professional translator, are your persional consideration that not follow base line of italian grammar.
Then current translation in many points can be considered a simple translation but not a translation that follow the base line of italian grammar.

I posted point by point the typical errors in current italian translation.
As I wrote the current italian translation has points where seems a word to word translation by english that not include the mandatory changes that are related to italian grammar.

@TurboGit 

If for italian language about your application it's necessary to post the changes in another github repository pelase add this important info in home page of your repsoioptry.
It's quite frustrating din't find this info, modify the file, and after that I discovered that I should load/modify the file from another repository.

**Comment by TurboGit** [comment]
@bovirus : It is not mandatory to use a separate github repository so there is nothing to document really, but people are free to organize translations as they want. Here the Italian team as decided to use a separate GitHub to discuss, review and coordinate the translation to ensure a consistent wording. I have also learned this at the same time as you :) In general it is always good to contact previous translator(s), that's why when I saw your first PR here I have pinged @mmardegan as I certainly cannot asses about the quality of the translations myself (for obvious reasons).

**Comment by bovirus** [comment]
@TurboGit 

I'm not agree with you.

The rule for github if you don't make  special mention in home page of repository (for x language there is a special different repository) is that anyone can make a PR for language.

If the user before to post a new language version should post in another place you have to mention in your repository page. 

If you didn't do the risk (as what's happended) that I spent my time for nothing (translate something that is not the last version and the new version is on a different repostory that is not reported)..

And is not a good service that you offer to the translators.

I'm a professional translator that woks for many freeware/shareware software projects (many on github).

This the first time that I see a open project organized in so strange way and with missing documentation about it.

If the documenatton was correct I will not spent time for nothing.

Thanks.

**Pull request closed without merging**